src/rotary-dialer-element.ts

```diff
@@ -0,0 +1,280 @@
+import { css, customElement, html, LitElement, property } from 'lit-element';
+
+enum Digit {
+  zero,
+  one,
+  two,
+  three,
+  four,
+  five,
+  six,
+  seven,
+  eight,
+  nine,
+}
+
+enum Direction {
+  normal = 'normal',
+  reverse = 'reverse',
+}
+@customElement('wokwi-rotary-dialer')
+export class RotaryDialerElement extends LitElement {
+  @property() value: number | '' = 0;
+
+  // todo: the degrees increases in ~40 when counting up the digits, would do that reusable if I could tweak the @keyframe from js
```

Comment:
Do you need any help with that?

Reply:
I actually found how to do that by insert a css rules but I don't think that's a "way to go" what do you think? see this as a reference --> https://stackoverflow.com/questions/18481550/how-to-dynamically-create-keyframe-css-animations

Reply:
See my suggestions below - perhaps using a CSS variable, or using a CSS `transition` instead will help here